CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ option(SOCI_STATIC "Enable build of static libraries" ON)
 option(SOCI_TESTS "Enable build of collection of SOCI tests" ON)
 option(SOCI_ASAN "Enable address sanitizer on GCC v4.8+/Clang v 3.1+" OFF)
 option(SOCI_LTO "Enable link time optimization" OFF)
+option(SOCI_VIS, "Enable symbol trimming via C++ visibility: Preset visibility to hidden" off)
```

Comment:
Hmm, I would be _for_ setting this one to `ON` by default, why not? I.e. I don't think there are any drawbacks to doing it.

Also, let's use something more readable, even if longer, for the option name, e.g. `SOCI_VISIBILITY`.

Reply:
Done